stablehlo/reference/Ops.cpp

```diff
@@ -1248,10 +1325,9 @@ SmallVector<Tensor> evalSortOp(ArrayRef<Tensor> inputs, Axis dimension,
 
     // Instead of literally putting the slices together into a vector of tuples,
     // we're representing these tuples with integer handles, with each handle
-    // being an index within the slice.
-    // Then, instead of sorting a vector of tuples, we're sorting a vector of
-    // handles, and the comparator knows how to use these handles to fetch
-    // the actual input elements being compared.
+    // being an index within the slice. Then, instead of sorting a vector of
+    // tuples, we're sorting a vector of handles, and the comparator knows how
+    // to use these handles to fetch the actual input elements being compared.
```

Comment:
Is there a particular reason why the formatting needs to be changed here? I think the previous version is a bit easier to read.

Reply:
Agreed, changed back.